osbrain/tests/test_agent.py

```diff
@@ -599,16 +604,17 @@ def test_running_exception(nsproxy):
     assert not agent.is_running()
 
 
+@skip_windows_port_reuse
 def test_agent_error_address_already_in_use(nsproxy):
     """
     Running an agent should raise an error if address is already in use.
     """
     with pytest.raises(RuntimeError) as error:
         run_agent('a0', nsaddr=nsproxy.addr(), addr=nsproxy.addr())
     assert 'OSError' in str(error.value)
-    assert 'Address already in use' in str(error.value)
```

Comment:
You removed this line but now that we are skipping for Windows, it should be there.